.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,4 @@
 - [ ] The page has 8 or fewer examples.
 - [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
 - [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
+- [ ] The page includes a web link (if applicable)
```

Comment:
I would further clarify that if it is a core linux util, it doesn't need a web link.

Reply:
I actually wouldn't object to adding a link to a html rendering of the manpage (in a reasonably canonical location), or GNU manual, etc. for such commands.

Reply:
I think linking to a man page would be pretty redundant. The whole reason why we started adding homepage links was that folks couldn't find further info about a command. But for core util commands, everything is just a `man <command>` away. Thus, I feel adding a homepage link for core linux utils does not make much sense.

Reply:
I agree with @agnivade here.

Reply:
@agnivade if nothing else is available for a given tool, would you prefer having no link at all instead of having a link to the manpage?

Reply:
Yes, only if the tool is a core linux util. Having links to manpages for `cp`, `mv` is pretty redundant IMO.

Reply:
Well, remember that tldr pages are not necessarily consumed on a command line. There's ostera's web client, the duckduckgo integration, the pages at distrowatch, mobile clients, etc. I think _not including any link at all_ based on the assumption that the user is looking at the tldr page in a command line is doing a disservice to users. Of course, if better links exist, I fully suport using them other than the manpage.

In other words, in my view every page should have a link of some sort, even if just to confirm the provenance of the tool.

Reply:
I get your point. You want the user to readily have a link to the full documentation. I prefer to have the links only in cases where it is not dead obvious where to look for documentation. It doesn't matter whether the user is in command line or not, they can always search for "man cp" and get the manpage.

That's just my opinion. I will leave it to the community to come to a decision.

Reply:
Fair enough. My position is that it's better to err on the side of offering unneeded help than not offering needed help. But let's flesh this out at #3041.